Data/Sys/Profiles/Wiimote/SDL Sideways Wiimote.ini

```diff
@@ -0,0 +1,50 @@
+[Profile]
```

Comment:
I think all the file names should say `Wii Remote` instead of `Wiimote`.
We are trying to use the proper terminology in user-facing strings now.